assembly/src/tokens/mod.rs

```diff
@@ -255,7 +285,12 @@ impl<'a> Token<'a> {
         assert_eq!(invocation_token, self.parts[0], "not an {invocation_token}");
         match self.num_parts() {
             0 => unreachable!(),
-            1 => Err(ParsingError::missing_param(self)),
+            1 => Err(
+                    ParsingError::missing_param(
+                        self,
+                        "{invocation_token_string} <invocation_token>"
+                    )
+                ),
```

Comment:
I don't think this will work correctly as `invocation_token_string` is not defined anywhere above. I'd probably change this to something like: `format!("{invocation_token}.<procedure_name>")`.

Reply:
Changed.